starters/shopify-next-tailwind/.env.example

```diff
@@ -1,6 +1,6 @@
-SESSION_SECRET="foobar"
-STOREFRONT_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-STOREFRONT_API_SECRET=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-PUBLIC_STOREFRONT_API_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-PUBLIC_STOREFRONT_API_VERSION=2023-04
-PUBLIC_STORE_DOMAIN=xxxxxxxxxxx.myshopify.com
+SESSION_SECRET="op://Internal Shopify/env/prod/SESSION_SECRET"
+STOREFRONT_API_KEY="op://Internal Shopify/env/prod/STOREFRONT_API_KEY"
+STOREFRONT_API_SECRET="op://Internal Shopify/env/prod/STOREFRONT_API_SECRET"
+PUBLIC_STOREFRONT_API_TOKEN="op://Internal Shopify/env/prod/PUBLIC_STOREFRONT_API_TOKEN"
+PUBLIC_STOREFRONT_API_VERSION="op://Internal Shopify/env/prod/PUBLIC_STOREFRONT_API_VERSION"
+PUBLIC_STORE_DOMAIN="op://Internal Shopify/env/prod/PUBLIC_STORE_DOMAIN"
```

Comment:
Since we are working on starter.dev and this will be the env example file, I think we would like to revert this change and leave the file with `xxxx` characters.